### DISCUSSION TITLE
Mixing Multi input of my soundcard to blackhole

### DISCUSSION THREAD

**Discussion by Balian38** [ExistentialAudio/BlackHole#723] [Q&A]
Hello
Il have a Scarlett solo with one mic in the first entry and my guitar plugin in the second one.
When I try ton create a mutioutput with black hole in my daw (garageband), and I make a skype conversation, the other one car hear only one track (the mic or the guitar). How can I mix all of tracks to output.

Thanks

PS : i 'm French so my English is poor !! ;)

**Comment by devinroth** [comment]
I’m working on an app that will do that. Called  Dipper. You can test it here. https://discord.gg/rDzxuWha3H

Devin
